csharp/Platform.Data.Doublets/Decorators/LinksDecoratorBase.cs

```diff
@@ -109,42 +109,42 @@ protected LinksDecoratorBase(ILinks<TLink> links) : base(links)
         /// <para>The handler.</para>
         /// <para></para>
         /// </param>
-        /// <param name="restrictions">
-        /// <para>The restrictions.</para>
+        /// <param name="restriction">
+        /// <para>The restriction.</para>
         /// <para></para>
         /// </param>
         /// <returns>
         /// <para>The link</para>
         /// <para></para>
         /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public virtual TLink Each(Func<IList<TLink>, TLink> handler, IList<TLink> restrictions) => _links.Each(handler, restrictions);
+        public virtual TLink Each(IList<TLink> restriction, Func<IList<TLink>, TLink> handler) => _links.Each(restriction, handler);
```

Comment:
May be
```suggestion
        public virtual TLink Each(IList<TLink> restriction, Handler<TLink> handler) => _links.Each(restriction, handler);
```

Reply:
Yes, we can define ReadHandler and WriteHandler in separate files.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/delegates/how-to-declare-instantiate-and-use-a-delegate